### PR TITLE
build: fix version handling

### PR DESCRIPTION
**Pull request by AlexNPavel**
The old Makefile had a typo that caused the output binaries to
have the version unchanged ("ct was not built properly").
This also removes the build script as it was unneccessary and
update the test file, which previously depended on that script.

### CODE REVIEW

File: test [hunk 1]
```diff
@@ -1,35 +1,28 @@
-#!/usr/bin/env bash
```
Comment (crawford) [inline test]: Use `/usr/bin/env bash`. Not all distros keep bash in `/bin`.

File: test [hunk 2]
```diff
@@ -1,35 +1,28 @@
...
+#!/bin/bash -e
```
Comment (dghubble) [inline test]: I'd keep the old version of this line. Also don't need to have -e and then set -e right below.

File: test [hunk 3]
```diff
@@ -1,35 +1,28 @@
...
 
-PKG=$(cd gopath/src/${REPO_PATH}; go list ./... | \
-	grep --invert-match vendor)
-
-# https://github.com/golang/go/issues/15067
-PKG_VET=$(cd gopath/src/${REPO_PATH}; go list ./... | \
-	grep --invert-match vendor | \
-	grep --invert-match internal/log)
+PKGS=$(go list ./... | grep -v /vendor)
+FORMATTABLE=$(ls -d */ | grep -v -E '(vendor/|examples/)')
```
Comment (dghubble) [inline test]: You can remove the exclusion of examples since there are is no examples folder in this project.

File: test [hunk 4]
```diff
@@ -1,35 +1,28 @@
...
 
 echo "Checking gofix..."
 go tool fix -diff $SRC
 
 echo "Checking gofmt..."
```
Comment (dghubble) [inline test]: were you going to add an issue or TODO to add `golint` checking? Its the only one missing

**Pull request merged**